pkgs/development/libraries/vtk/generic.nix

```diff
@@ -35,25 +33,11 @@ in stdenv.mkDerivation {
       xorgproto
       libXt
     ] ++ optionals stdenv.hostPlatform.isDarwin [
-      xpc
-      AGL
-      Cocoa
-      CoreServices
-      DiskArbitration
-      IOKit
-      CFNetwork
-      Security
-      ApplicationServices
-      CoreText
-      IOSurface
-      ImageIO
-      OpenGL
-      GLUT
+        apple-sdk
```

Comment:
This does nothing; `apple-sdk` is already part of the standard environment. It can be omitted.